decidim-assemblies/app/cells/decidim/assemblies/assembly_cell.rb

```diff
@@ -11,8 +11,14 @@ def show
 
       private
 
+      # REDESIGN_DETAILS: size :m will be deprecated
       def card_size
```

Comment:
This could be simplified using a guard method, but i will let it pass though to ensure consistency across multiple cells where you render the size: :s

Reply:
Yes, I have kept the same that exists in other cells